### PR TITLE
get-started: update Get Started accord to most recent example-get-started repo

### LINKED ISSUE #487: cmd ref: advanced examples for import (and get)
> The previous PR was too long. We're extracting this part, which is pending and hopefully the last related to `dvc import` and `dvc get`. See https://github.com/iterative/dvc.org/pull/464#pullrequestreview-259871929

- [x] Note that the `import-url` ref and `external-dependencies` guide also needs an update (See https://github.com/iterative/dvc.org/pull/464#pullrequestreview-257721375).

- [x] Create more sample repos for advanced `import` and possibly `get` command example/s. See https://github.com/iterative/dvc.org/issues/487#issuecomment-517520838 below.

- [x] Actually create/update advanced examples for import/get commands.

- ~~Check the `example-pipeline` carefully. It looks like it might be broken now. (Check both examples in fact.)~~ Moved to #564

- [x] Lets also update get started, work on use cases, etc. to incorporate these important new commands (include also `dvc update`).

### LINKED ISSUE #554: lint: inconsistent fenced code block `-`line indentation
A problem after implementing #372 is that some lines inside md fenced code blocks starting with `-` get indented for unknown reasons by Prettier.

E.g. by finding the regex `^- cache` in all files we can see that static/docs/commands-reference/commit.md has the unindented lines:

image
> Similar cases in static/docs/get-started/connect-code-and-data.md and static/docs/tutorial/define-ml-pipeline.md

However static/docs/commands-reference/add.md has them indented:

image

I don't see a clear pattern explaining this apparently erratic behavior but it's all over the docs.

Again, this i work of Prettier which has been run on all md files in #512. None of the DVC-file samples had indentation before `-` lines originally. We decided that the indentation was OK in https://github.com/iterative/dvc.org/issues/512#issuecomment-518878493 but the fact that it indents some and unindents others troubles me a little. Perhaps we should use `<!-- prettier-ignore -->`s before all YAML code blocks after all?

All these code samples are still valid YAML though so please just close this issue if the decision is to live with it and let `prettier` do whatever it wants 😋 

---

Originally reported partially as [lint: don't indent fenced code block contents](https://github.com/iterative/dvc.org/issues/512)

### PR DESCRIPTION
**Pull request by jorgeorpinel**
Creating https://github.com/iterative/example-repos-dev and derivate repos.

Review app docs @ https://dvc-org-pr-555.herokuapp.com/

---

Also fix #554 (See https://github.com/iterative/dvc.org/pull/555#issuecomment-520871308)

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Note: the last couple commits here are to fix #554

**Comment by jorgeorpinel** [comment]
Review app docs @ https://dvc-org-pr-555.herokuapp.com/

File: static/docs/get-started/connect-code-and-data.md [hunk 1]
```diff
@@ -48,17 +48,19 @@ recommend creating a virtual environment with a tool such as
 $ virtualenv -p python3 .env
 $ source .env/bin/activate
 $ echo ".env/" >> .gitignore
-$ pip install -r requirements.txt
+$ pip install -r src/requirements.txt
```
Comment (shcheklein) [inline static/docs/get-started/connect-code-and-data.md]: check other places in command references where we use example-get-started repo, sometimes we provide instruction on how to install it in the details section

File: static/docs/get-started/connect-code-and-data.md [hunk 2]
```diff
@@ -48,17 +48,19 @@ recommend creating a virtual environment with a tool such as
...
 Save the progress to Git:
 
 ```dvc
 $ git add .
-$ git commit -m "add code"
+$ git commit -m "Add source code files to repo"
 ```
 
-</details>
+## Create a first data transformation stage
```
Comment (shcheklein) [inline static/docs/get-started/connect-code-and-data.md]: I would avoid using second level titles. If there is a temptation to use it it means that section has become too complicated. The ideal size of the chapter for the get started - one screen. It should be simple, just enough to give some sense of the idea.

File: static/docs/get-started/connect-code-and-data.md (hunk 1 above)
Comment (jorgeorpinel) [reply static/docs/get-started/connect-code-and-data.md]: Another good catch, thanks! Fixed in 9885f13

File: static/docs/get-started/connect-code-and-data.md (hunk 2 above)
Comment (jorgeorpinel) [reply static/docs/get-started/connect-code-and-data.md]: Updated in 60a998b per chat IRL.

**Pull request merged**